### PR TITLE
Rename `MaxNumberOfCosmicClusters`  parameter in  `ClusterChecker`

### PR DESCRIPTION
**Pull request by mmusich**
#### PR description:

This PR is purely technical, rename `MaxNumberOfCosmicClusters`  into `MaxNumberOfStripClusters` in `ClusterChecker` as it is currently misleading and still there only for "historical reasons" (see https://github.com/cms-sw/cmssw/commit/a551c6ea3c8023c2704b5e4fe63f12983378123e). 
Some gymnastics is required in order to apply the change in the existing HLT configurations.
No regressions are expected.

#### PR validation:

Run successfully:
   * `addOnTests.py`
   * `runTheMatrix.py -l limited --ibeos`

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

 Not a backport, no backport is needed.

### CODE REVIEW

**Comment by mmusich** [comment]
please test

**Comment by mmusich** [comment]
@cmsbuild, please test
   * looks like a glitch in eos / xrootd, let's try again

File: HLTrigger/Configuration/python/customizeHLTforCMSSW.py [hunk 1]
```diff
@@ -247,6 +247,20 @@ def customizeHLTfor42497(process):
...
+    for prod in producers_by_type(process, 'ClusterCheckerEDProducer'):
+        if hasattr(prod, "MaxNumberOfCosmicClusters"):
+            prod.MaxNumberOfStripClusters = getattr(prod,"MaxNumberOfCosmicClusters")
+            prod.__delattr__("MaxNumberOfCosmicClusters")
+
+    for prod in producers_by_type(process, 'SeedGeneratorFromRegionHitsEDProducer'):
+        if hasattr(prod, "ClusterCheckPSet"):
+            clustCheckPSet = getattr(prod,"ClusterCheckPSet")
+            prod.ClusterCheckPSet.MaxNumberOfStripClusters = getattr(clustCheckPSet,"MaxNumberOfCosmicClusters")
+            clustCheckPSet.__delattr__("MaxNumberOfCosmicClusters")
```
Comment (jfernan2) [inline HLTrigger/Configuration/python/customizeHLTforCMSSW.py:261]: Shouldn't this be "MaxNumberOfStripClusters" ?

Comment (mmusich) [reply HLTrigger/Configuration/python/customizeHLTforCMSSW.py:261]: No.

**Comment by jfernan2** [comment]
+1

**Comment by civanch** [comment]
+1

**Comment by mmusich** [comment]
+hlt

**Comment by mmusich** [comment]
@cms-sw/dqm-l2 your signature is required here

**Comment by tjavaid** [comment]
+1

**Comment by antoniovilela** [comment]
+1

**Pull request merged**